Formula/code-server.rb

```diff
@@ -14,7 +14,7 @@ class CodeServer < Formula
 
   depends_on "python@3.9" => :build
   depends_on "yarn" => :build
-  depends_on "node"
+  depends_on "node@12"
```

Comment:
```suggestion
  depends_on "node@14"
```
Can we use `node@14` here, because that's the LTS version?